app/code/Elogic/Provider/Api/ProviderRepositoryInterface.php

```diff
@@ -1,46 +1,48 @@
 <?php
-
+/**
+ * Copyright © Bohdan Rakochyi, Inc. All rights reserved.
```

Comment:
example of copyright
/**
 * Elogic provider repository
 *
 * @category  Elogic
 * @package   Elogic\Provider
 * @author    Bohdan Rakochyi <you work EMAIL>
 * @copyright 2021 Elogic
 */